FRAuth/FRAuth/DeviceBinding/ApplicationPinDeviceAuthenticatorDelegate.swift

```diff
@@ -0,0 +1,72 @@
+// 
+//  ApplicationPinDeviceAuthenticatorDelegate.swift
+//  FRAuth
+//
+//  Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import UIKit
+
+
+/// ApplicationPinDeviceAuthenticatorDelegate for collecting the Pin
+public protocol ApplicationPinDeviceAuthenticatorDelegate: AnyObject {
+    /// Delegate method to collect the Pin
+    func collectPin(prompt: Prompt, completion: @escaping (String?) -> Void)
+}
+
+
+/// Default implementation for ApplicationPinDeviceAuthenticatorDelegate
+public class ApplicationPinDeviceAuthenticatorDefaultDelegate: NSObject, ApplicationPinDeviceAuthenticatorDelegate {
```

Comment:
Same as above that would then be the default collector

`public class ApplicationPinDeviceAuthenticatorDefaultCollector: NSObject, ApplicationPinDeviceAuthenticatorCollectorDelegate {`

Reply:
+1

Reply:
We decided on **DefaultPinCollector**